app/js/app/services/SearchResults.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2018 Ben Hanson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+define([], function() {
+	let searchList = [
+		{
+			id: "assignments",
+			title: "My Assignments",
+			terms: ["my assignments", "assignments", "homework", "assignemts"],
+			url: "/assignments",
+			type: "shortcut"
+		}, {
+			id: "join_group",
```

Comment:
I'd make the title of this "Teacher Connections" for consistency, and if possible add a subtitle like:
"Join groups and manage teacher connections."